From daeffac738603a9a673bd6827f827b591e282399 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Thu, 15 Sep 2016 21:18:57 +0100
Subject: [PATCH 09/12] applets/status: Absolutely require a minimum of 0.99.0
 for upower

We're not going to maintain compatibility for very old upower versions as
we cannot deploy tarballs and have #ifdef's all over the place, as we're
bound by vapi.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 panel/applets/status/configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panel/applets/status/configure.ac b/panel/applets/status/configure.ac
index eece1ee..a990dbc 100644
--- a/panel/applets/status/configure.ac
+++ b/panel/applets/status/configure.ac
@@ -1,5 +1,5 @@
 m4_define([accountsservice_version], [0.6.40])
-m4_define([upower_required_version], [0.9.20])
+m4_define([upower_required_version], [0.99.0])
 m4_define([bt_required_version], [3.16.0])
 
 PKG_CHECK_MODULES(STATUS_PLUGIN,
-- 
2.10.0

